frontend/src/citizen-frontend/calendar/reservation-modal/utils.ts

```diff
@@ -0,0 +1,76 @@
+// SPDX-FileCopyrightText: 2017-2022 City of Espoo
+//
+// SPDX-License-Identifier: LGPL-2.1-or-later
+
+import uniqBy from 'lodash/uniqBy'
+
+import {
+  DailyReservationData,
+  TimeRange
+} from 'lib-common/generated/api-types/reservations'
+import { TimeRanges } from 'lib-common/reservations'
+
+export const emptyTimeRange = [
+  {
+    startTime: '',
+    endTime: ''
+  }
+] as [TimeRange]
```

Comment:
You shouldn't use `as [TimeRange]` because it basically turns off type checking. If you want `emptyTimeRange` to be some other type other than what was inferred, you can use a type annotation: `export const emptyTimeRange: [TimeRange] = ...`.